gui/storage/views.py

```diff
@@ -408,9 +408,11 @@ def dataset_create(request, fs):
             recordsize = cleaned_data.get('dataset_recordsize')
             if recordsize:
                 props['recordsize'] = recordsize
+            dataset_comments = cleaned_data.get('dataset_comments')
             errno, errmsg = notifier().create_zfs_dataset(
                 path=str(dataset_name),
                 props=props)
+            notifier().zfs_set_option(name = str(dataset_name),item = "storage:comments",value = dataset_comments)
```

Comment:
Like I said in previous PR, thats not PEP8, this is:
name=str(dataset_name), item="storage:comments", value=dataset_comments